sdk/translation/Azure.AI.Translation.Document/tests/UnitTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.Translation.Document.Tests
+{
+    /// <summary>
+    /// Unit tests for our utility methods and helpers.
+    /// </summary>
+    public class UnitTests
```

Comment:
looking into our current tests, you could actually just add the test to the class https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/translation/Azure.AI.Translation.Document/tests/DocumentTranslationClientTests.cs

Add a method per function that you added the validation to. Each method will be something like:
```
var expectedType = myId == null
                ? typeof(ArgumentNullException)
                : typeof(ArgumentException);

Assert.Throws(expectedType,  =>  new DocumentTranslationOperation(translationId, client);

```

and you can have like 3 `TestCases` like: 
```
[TestCase(null)]
[TestCase("Foo Bar")]
[TestCase("")]
```

This is all based on: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientTests.cs#L336